bundle/config/experimental.go

```diff
@@ -45,6 +45,12 @@ type PyDABs struct {
 	// These packages are imported to discover resources, resource generators, and mutators.
 	// This list can include namespace packages, which causes the import of nested packages.
 	Import []string `json:"import,omitempty"`
+
+	// LoadLocations is a flag to enable loading Python source locations from the PyDABs.
+	//
+	// Locations are only supported since PyDABs 0.6.0, and because of that,
```

Comment:
Shouldn't it just try to load locations and fail gracefully?

Reply:
We will enable it by default when we move out of the `experimental` section. It will be extra difficult to handle the case when the subprocess fails with an unknown argument error and not another error because we will need to parse stderr for that. Not sure it's worth it.

Reply:
Newer versions will ignore and warn about unknown flags. We should have done it from the beginning.

Reply:
@kanterov Could you revisit this for Jobs as Code? Seems like we basically have a clean slate.

And please update this and the types etc. to get rid of the "PyDABs" reference.